shelley/chain-and-ledger/formal-spec/protocol-parameters.tex

```diff
@@ -7,15 +7,25 @@ \section{Protocol Parameters}
 and $\mathsf{T_{ppm}}$ represents the dependent type of a given protocol parameter.
 The parameters are listed in Figure~\ref{fig:defs:protocol-parameters},
 the first ten of which below are common to the Byron era.
+(NSF: I'm not sure the first ten are common to Byron. If they are, it's not
+obvious to me where to find their counterparts. I'm primarily merely suggesting
+to double-check that sentence, in case it has gone stale.)
 We will explain the significance of each parameter as it comes up in
 the calculations used in transition rules.
 
 The type $\Coin$ is defined as an alias for the integers.
 Negative values will not be allowed in UTxO outputs or reward accounts,
 and $\Z$ is only chosen over $\N$ for its additive inverses.
+(NSF: I'm curious about an example or two where negative values are allowed?
+Is it just ephemerally in a some equations throughout?)
 
 Eight global constants are defined.
 As global constants, these values can only be changed by updating the software.
+(NSF: That sentence has me immediately concerned about nodes running different
```

Comment:
I am not sure @nfrisby .. I think running old versions of software is OK with old constants. But it seems like it should not be?